From 6d8243aec194b40f46c8d23ec386cd1af7590b4a Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 21 Jul 2022 20:44:25 +0200
Subject: [PATCH 1/2] look for conda-forge variant of libwebp

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f3cb86e..8ba4b6d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -687,7 +687,7 @@ if(SDL2IMAGE_WEBP)
         add_library(SDL2_image::external_libwebp ALIAS webp)
     else()
         message(STATUS "${PROJECT_NAME}: Using system libwebp")
-        find_package(webp REQUIRED)
+        find_package(WebP REQUIRED)
         list(APPEND PC_REQUIRES libwebp)
     endif()
     if(SDL2IMAGE_WEBP_SHARED)
