diff --git a/CMakeLists.txt b/CMakeLists.txt
index 71a85f7..4792bac 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -683,7 +683,6 @@ if(SDL2IMAGE_INSTALL)
             "${CMAKE_CURRENT_BINARY_DIR}/SDL2_imageConfig.cmake"
             "${CMAKE_CURRENT_BINARY_DIR}/SDL2_imageConfigVersion.cmake"
             cmake/Findlibjxl.cmake
-            cmake/Findwebp.cmake
         DESTINATION "${PKG_PREFIX}"
         COMPONENT devel
     )
diff --git a/cmake/Findwebp.cmake b/cmake/Findwebp.cmake
deleted file mode 100644
index 23982dd..0000000
--- a/cmake/Findwebp.cmake
+++ /dev/null
@@ -1,32 +0,0 @@
-include(FindPackageHandleStandardArgs)
-
-find_library(webp_LIBRARY
-    NAMES webp
-)
-
-find_path(webp_INCLUDE_PATH
-    NAMES webp/decode.h
-)
-
-set(webp_COMPILE_OPTIONS "" CACHE STRING "Extra compile options of webp")
-
-set(webp_LINK_LIBRARIES "" CACHE STRING "Extra link libraries of webp")
-
-set(webp_LINK_FLAGS "" CACHE STRING "Extra link flags of webp")
-
-find_package_handle_standard_args(webp
-    REQUIRED_VARS webp_LIBRARY webp_INCLUDE_PATH
-)
-
-if (webp_FOUND)
-    if (NOT TARGET WebP::webp)
-        add_library(WebP::webp UNKNOWN IMPORTED)
-        set_target_properties(WebP::webp PROPERTIES
-            IMPORTED_LOCATION "${webp_LIBRARY}"
-            INTERFACE_INCLUDE_DIRECTORIES "${webp_INCLUDE_PATH}"
-            INTERFACE_COMPILE_OPTIONS "${webp_COMPILE_FLAGS}"
-            INTERFACE_LINK_LIBRARIES "${webp_LINK_LIBRARIES}"
-            INTERFACE_LINK_FLAGS "${webp_LINK_FLAGS}"
-        )
-    endif()
-endif()
